eisbuk-admin/src/components/atoms/AttendanceSheet/PrintButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import IconButton from "@material-ui/core/IconButton";
+
+import PrintIcon from "@material-ui/icons/Print";
+import { SlotButtonProps } from "@/types/components";
+
+import {
+  __slotButtonNoContextError,
+  __copyButtonWrongContextError,
+  __noDateCopy,
+} from "@/lib/errorMessages";
+
+import { __copyButtonId__ } from "@/__testData__/testIds";
+import { Link } from "react-router-dom";
+
+export const PrintButton: React.FC<SlotButtonProps> = ({ size }) => {
+  return (
+    <>
+      <IconButton component={Link} to="/attendance_printable" size={size}>
+        <PrintIcon />
+      </IconButton>
+    </>
+  );
+};
```

Comment:
This is just a copy/paste of `CopyButton` and most of the stuff is not neccessary, as the linter reports (a lot of unused imports). Normally I don't have anything against copying existing code and just tweaking the stuff you need (for efficiency), but make sure you completely adapt the new code (remove all of the stuff not necessary). 
Furthermore, there's no need for a separate component as this button should only be used in attendance page, so these three lines of code can easily be written in `@/pages/root/index.ts`.

Additional comment (with no relevance for the case at hand): if you were to use this as is, the react fragment is unnecessary since you're returning only one JSX.Element (IconButton) and everything else is nested inside.